build/documentation/generate-typedocs.js

```diff
@@ -80,7 +80,7 @@ function execute() {
 
     const packages = path.resolve(rootDir, srcDir);
 
-    glob(packages, { realpath: true }, function(error, srcFiles) {
+    glob(packages, { realpath: true }, function (error, srcFiles) {
```

Comment:
is the space after function necessary?  Looks like it might not be consistently applied?  Should this also apply to any function?

Reply:
Same note here on this being from prettier: https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot